src/astro/sql/__init__.py

```diff
@@ -59,6 +84,34 @@ def run_raw_sql(
     handler: Optional[Callable] = None,
     **kwargs: Any,
 ) -> TaskDecorator:
+    """
+    Given a python function that returns a SQL statement and (optional) tables, execute the SQL statement and output
+    the result into a SQL table.
+
+    Use this function as a decorator like so:
+
+      @transform
+      def my_sql_statement(table1: Table) -> Table:
+          return "DROP TABLE {{table1}}"
```

Comment:
We need to use rst code blocks for this to render as code:

<img width="721" alt="image" src="https://user-images.githubusercontent.com/8811558/175363390-42bdb0e6-6dd0-4511-b314-e27714a6b36e.png">

Example:

https://github.com/apache/airflow/blob/main/airflow/providers/google/cloud/operators/gcs.py#L90-L99